runtime/jvmti/jvmtiClass.c

```diff
@@ -1018,6 +1018,43 @@ fixWatchedFields(J9JavaVM *vm, J9HashTable *classHashTable)
 }
 
 
+static void
```

Comment:
Can you add a comment describing this here?

Reply:
Done.